CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.14)
 project(dwarf_bench)
 
+find_package(TBB REQUIRED)
```

Comment:
Due to the fact that `TBBSort` does not require `dpcpp`, I thought it would be better to separate `tbb` from `dpcpp` and be able to use `TBBSort` regardless of the presence of `sycl`
Is it a bad idea?

Reply:
No, sounds fine.